modules/amplify/predict/meta.yml

```
@@ -0,0 +1,44 @@
+name: "amplify_predict"
+description: AMPlify is an attentive deep learning model for antimicrobial peptide prediction.
+keywords:
+  - antimicrobial peptides
+  - AMPs
+  - prediction
+  - model
+tools:
+  - "amplify":
+      description: "Attentive deep learning model for antimicrobial peptide prediction"
+      homepage: "https://github.com/bcgsc/AMPlify"
+      documentation: "https://github.com/bcgsc/AMPlify"
+      tool_dev_url: "https://github.com/bcgsc/AMPlify"
+      doi: "https://doi.org/10.1186/s12864-022-08310-4"
+      licence: "['GPL v3']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - faa:
+      type: file
+      description: amino acid sequences fasta
+      pattern: "*.{faa,fasta}"
```

Nextflow typically handles gzip compressed files. If AMPlify cannot use those directly, a decompression step should be added.

```suggestion
      pattern: "*.{faa,faa.gz,fasta,fasta.gz}"
```